Dockerfile

```diff
@@ -14,24 +21,30 @@ RUN apt-get update \
       pip
 
 # Custom entrypoint
-COPY entrypoint.sh /usr/local/bin/entrypoint.sh
+COPY scraper/entrypoint.sh /usr/local/bin/entrypoint.sh
 ENTRYPOINT ["entrypoint.sh"]
 RUN mkdir -p /output
 WORKDIR /output
 
 # Copy pyproject.toml and its dependencies
-COPY pyproject.toml README.md openzim.toml /src/
-COPY src/youtube2zim/__about__.py /src/src/youtube2zim/__about__.py
+COPY README.md /src/
+COPY scraper/pyproject.toml scraper/openzim.toml /src/scraper/
+COPY scraper/src/youtube2zim/__about__.py /src/scraper/src/youtube2zim/__about__.py
 
 # Install Python dependencies
-RUN pip install --no-cache-dir /src
+RUN pip install --no-cache-dir /src/scraper
 
 # Copy code + associated artifacts
-COPY src /src/src
+COPY scraper/src /src/scraper/src
 COPY *.md LICENSE CHANGELOG /src/
 
 # Install + cleanup
-RUN pip install --no-cache-dir /src \
- && rm -rf /src
+RUN pip install --no-cache-dir /src/scraper \
+ && rm -rf /src/scraper
+
+# Copy zimui build output
+COPY --from=zimui /src/dist /src/zimui
+
+ENV KOLIBRI_ZIMUI_DIST=/src/zimui
```

Comment:
Same remark ^^

Reply:
Fixed it in [304e7c3](https://github.com/openzim/youtube/pull/207/commits/304e7c367e9ff184ea5865ae1499f880115766d4)